app/screens/send-bitcoin-screen/send-bitcoin-screen.tsx

```diff
@@ -60,6 +61,14 @@ export const SendBitcoinScreen: ScreenType = ({
 
   const [invoiceError, setInvoiceError] = useState("")
   const [address, setAddress] = useState("")
+  const [lnurl] = useState("")
+  const [minSendable, setMinSendable] = useState("")
+  const [maxSendable, setMaxSendable] = useState("")
+  const [lnurlDomain, setLnurlDomain] = useState("domain")
+  const [lnurlCb, setLnurlCb] = useState("")
+  const [lnurlError, setLnurlError] = useState("")
```

Comment:
We should probably combine all these states into one object as they're related and set together

Reply:
Also, please add proper TS types for this object. minSendable is a number but the code initialize it as a string here. TS will be confused.

Reply:
I converted these into a lnurlpay object, I'll add ts types next.

Reply:
done